styleguide.config.js

```diff
@@ -24,7 +28,7 @@ module.exports = {
             path.resolve(__dirname, 'src'),
             path.resolve(__dirname, 'logo'),
           ],
-          exclude: /node_modules/,
+          exclude: /(node_modules\/monaco-editor)/,
```

Comment:
We only want to exclude the monaco-editor node module now from this loader?

Reply:
ughh, good catch man